tensorboard/webapp/notification_center/_views/notification_center_test.ts

```diff
@@ -47,6 +47,8 @@ describe('notification center', () => {
       ],
     }).compileComponents();
     store = TestBed.inject<Store<State>>(Store) as MockStore<State>;
```

Comment:
Test for the interaction (bell button click) is missing.

Reply:
add 3 more tests, please help to check. Thanks!